src/gui/qgsprojectionselectiontreewidget.cpp

```diff
@@ -957,6 +1017,67 @@ long QgsProjectionSelectionTreeWidget::getLargestCrsIdMatch( const QString &sql
   return srsId;
 }
 
+void QgsProjectionSelectionTreeWidget::updateBoundsPreview()
+{
+  sqlite3      *database = nullptr;
+  const char   *tail = nullptr;
+  sqlite3_stmt *stmt = nullptr;
+
+  QTreeWidgetItem *lvi = lstCoordinateSystems->currentItem();
+  if ( !lvi || lvi->text( QgisCrsIdColumn ).isEmpty() )
+    return;
+
+  int result = sqlite3_open_v2( mSrsDatabaseFileName.toUtf8().data(), &database, SQLITE_OPEN_READONLY, nullptr );
```

Comment:
I don't think it should block this PR, but it'd be good to move this bounds retrieval to QgsCoordinateReferenceSystem itself. It'd be quite handy in custom scripts i'd imagine!